.ci/openshift_setup.sh

```diff
@@ -16,9 +16,14 @@
 
 set -e
 
-SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source /etc/os-release
-source "$SCRIPT_PATH/../test-versions.txt"
+
+# source versions.txt from clearcontainers/runtime repository
+runtime_repo="github.com/clearcontainers/runtime"
+if [ ! -f "$GOPATH/src/${runtime_repo}/versions.txt" ]; then
```

Comment:
Hi @chavafg , how about if `$GOPATH/src/${runtime_repo}/versions.txt` file exists, but the repo is old, should the repo do a `git pull` instead?

Reply:
That's a good point. For CI systems, this shouldn't happen as we create a new environment for every PR, but we probably should do this anyway for the developer system scenario. In fact, how about changing this PR slightly to reduce the boiler-plate code. We can "edge" slowly towards a [`manage_ctr_mgr.sh`](#666) by creating a script (called `foo.sh` below but call it whatever you think is appropriate at this stage) that is just:

```
set -e

_runtime_repo="github.com/clearcontainers/runtime"
go get -d -u -v "$_runtime_repo" || true

[ ! -f "$_versions_file" ] && { echo >&2 "ERROR: cannot find $_versions_file"; exit 1; }
_versions_file="$GOPATH/src/${_runtime_repo}/versions.txt"

source "$_versions_file"

unset _versions_file
unset _runtime_repo
```
Then, all the other scripts just need to do:

```
source foo.sh
```
... and this should DTRT (TM) :smile:

That will minimise the amount of code required in the calling scripts and also "hide" the location of `versions.txt` so that we can change it at will later if required with no impact on the callers. In fact, I wonder if it would be worth wrapping the guts of `foo.sh` into a function so it becomes something like:

```
set -e

_runtime_repo="github.com/clearcontainers/runtime"
_versions_file="$GOPATH/src/${_runtime_repo}/versions.txt"

get_cc_versions()
{
    go get -d -u -v "$_runtime_repo" || true

    [ ! -f "$_versions_file" ] && { echo >&2 "ERROR: cannot find $_versions_file"; exit 1; }

    source "$_versions_file"
}

unset _versions_file
unset _runtime_repo
```

and for the callers:

```
source foo.sh
get_cc_versions

# use $docker_version, etc
```

The advantage of that being we could start to introduce parameters for `get_cc_versions()` at a later date.

Reply:
I like the idea, will add the function to  `.ci/lib.sh`